editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,38 @@
 caption: Formatting
-created: 20131205155959399
-modified: 20220513120211686
+created: 20220727060921261
+modified: 20220821183525017
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
-Available character formatting includes:
+Available character formatting in WikiText includes:
 
-* <code>&#96;backticks&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>)
-** Alternatively, <code>&#96;&#96;double backticks allows &#96;embedded&#96; backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold text'' (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic text// (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored text__ (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)
+<div class="no-code-border">
+
+|Wikitext | Button |Win Shortcut | Mac |Rendered Output |h
+|2 single hyphens are used for `''bold text''`| <kbd><<.icon $:/core/images/bold>></kbd> | <kbd>ctrl-B</kbd> |<kbd>win-B</kbd> |2 single hyphens are used for ''bold text''|
```

Comment:
It seems confusing to include editor shortcuts here. Maybe it would make sense in the cheat sheet itself, but here it just seems out of place (we don't do anything similar in any of the other wikitext docs).

Reply:
>It seems confusing to include editor shortcuts here. 

I think, it's a big improvement. All the changes made for this PR have their origin in posts from Talk TiddlyWiki. There are a lot of complains about missing wikitext info for starters. 

>Maybe it would make sense in the cheat sheet itself, but here it just seems out of place 

There isn't enough space on mobile screens. In this tiddler it works, except keyboard shortcuts don't work on mobile. So they are read only there. 

>(we don't do anything similar in any of the other wikitext docs).

That's right. I think, we should add more info about keyboard shortcuts, where it belongs to. But having 10 tiddlers more in this PR I would probably need a year to get it merged. ... 

IMO it's the -docs branch. We should iterate fast here. If something doesn't work, removing it is simple. Adding and formatting costs a lot of time.

But we didn't even test it with real end-users. We didn't promote the new docs, so we didn't get any new feedback.  -- I personally don't need a single line of all this docs added here.